platforms.json

```diff
@@ -277,7 +277,16 @@
                 "\\$protobuf",
                 "important,protobuf",
                 "\\$extension",
-                ",extension"
+                ",extension",
+                "\\$jsinject",
+                ",jsinject",
+                "\\$urlblock",
+                ",urlblock",
+                "\\$content",
+                ",content",
+                "$webrtc",
+                "#@%#",
```

Comment:
`#@%#` will remove scriptlets exclusion rules, but we convert scriptlets rules to uBO syntax, so let's keep them.